src/ucs/vfs/base/vfs_obj.c

```diff
@@ -259,13 +259,21 @@ static ucs_status_t ucs_vfs_node_add(void *parent_obj, ucs_vfs_node_type_t type,
                                      va_list ap, ucs_vfs_node_t **new_node)
 {
     ucs_vfs_node_t *current_node;
```

Comment:
coding guideline mandates `current_node` be after initialized variables

Reply:
need to move `current_node` line 264

Reply:
done

Reply:
Outdated, done in other PRs